UnityProject/Assets/Scripts/Items/Stackable.cs

```diff
@@ -114,7 +116,7 @@ public void OnSpawnServer(SpawnInfo info)
 		InitStacksWith();
 		SyncAmount(amount, initialAmount);
 		amountInit = true;
-		ServerStackOnGround(registerTile.LocalPositionServer);
+		ServerStackOnGround(gameObject.RegisterTile().LocalPosition);
```

Comment:
Didn't need to change this
```suggestion
		ServerStackOnGround(registerTile.LocalPositionServer);
```